### DISCUSSION TITLE
Defining the CONCOURSE_TEAM for worker

### DISCUSSION THREAD

**Discussion by anandgupta0404** [concourse/concourse#8772] [General]
Hello,

How can we define the 2 team together in  CONCOURSE_TEAM environment variable? 

or what if I want to define 2 team for the 2 separate worker.

Thanks !

@taylorsilva @pnsantos @evanchaoli @jamieklassen

**Comment by ceco556** [comment]
Hello @taylorsilva, 

We have the same issue in our Concourse instance. We want to have specific workers for only two teams, but unfortunately this cannot happen as the variable is accepting only one single team.

Is there any reason for that or i can open a PR fixing this?

**Reply by taylorsilva** [reply]
Good find here! I can't think of any reason why we couldn't do this. I expect the PR to be a bit big, will end up touching a lot of layers in the code base (db, api, fly possibly to format output correctly?).

Go for it, will be a nice enhancement!

**Reply by ceco556** [reply]
Great, thank you!

I will take a look in the code and will try to open a PR next days.
